Geometry_Engine/Query/IsContaining.cs

```diff
@@ -353,6 +353,15 @@ public static bool IsContaining(this PolyCurve curve, List<Point> points, bool a
                 if (pPt.SquareDistance(pt) > sqTol) // not on the same plane
                     return false;
 
+                double distance = pPt.Distance(curve);
```

Comment:
Wow, this is pretty brute force 😉 Hard to believe that this had not been captured in the past?

Reply:
Yeah, I know.. 

A bit unsure if what I did in https://github.com/BHoM/BHoM_Engine/pull/2802 caused some issue with this particular edge case. I am not sure how my changes there would though...

anyway, I think this might actually be a performance improvement for some cases (points on edges) as it skips the need for intersections etc. Ofc, will be a slowdown for other cases.

Basically, this _is_ attempted to be checked for further down, but it dos fail for some weird edgecases.